google-chat-bot/src/main/java/io/spine/chatbot/server/google/chat/ThreadChatRepository.java

```diff
@@ -63,7 +64,8 @@
     protected void setupEventRouting(EventRouting<ThreadId> routing) {
         super.setupEventRouting(routing);
         routing.route(BuildFailed.class, new RepositoryEventRoute<>())
-               .route(BuildRecovered.class, new RepositoryEventRoute<>());
+               .route(BuildRecovered.class, new RepositoryEventRoute<>())
```

Comment:
Why do we need a class here at all? Could that be a single method of the `ThreadChatRepository` with a method reference here?

Reply:
Is it something to do with the Java serialization?

Reply:
Replaced with a method reference.